doc/vimspector.txt

```diff
@@ -167,10 +169,10 @@ Image: Build [3] Image: Matrix [5] Image: Gitter [7]
 
 - Debug profile configuration
 
-  - C, C++, Rust, etc.
+  - C, C , Rust, etc.
```

Comment:
_Gee Bill, two Cs?_

Reply:
Ugh. I forgot I had to fix that up. I’ll do it next time.